src/quo2/components/settings/settings_list/style.cljs

```diff
@@ -21,6 +21,12 @@
    :padding-horizontal 12
    :padding-vertical   13})
 
+(def border-style
+  {
```

Comment:
`border-style` should be a function because the map depends on a reactive atom. Also the braces are incorrectly formatted.